applications/connectivity_bridge/prj.conf

```diff
@@ -75,3 +75,6 @@ CONFIG_HW_ID_LIBRARY=y
 CONFIG_RESET_ON_FATAL_ERROR=y
 CONFIG_ASSERT=y
 CONFIG_POWEROFF=y
+
+# BT
+CONFIG_BT_BUF_CMD_TX_COUNT=3
```

Comment:
I think this should be moved to the .conf files in `boards` and be set along with the rest of the BT options